pkg/skaffold/test/structure/structure.go

```diff
@@ -46,11 +48,12 @@ func New(cfg docker.Config, tc *latestV1.TestCase, imageIsLocal bool) (*Runner,
 		return nil, err
 	}
 	return &Runner{
-		structureTests: tc.StructureTests,
-		imageName:      tc.ImageName,
-		workspace:      tc.Workspace,
-		localDaemon:    localDaemon,
-		imageIsLocal:   imageIsLocal,
+		structureTests:    tc.StructureTests,
+		imageName:         tc.ImageName,
+		workspace:         tc.Workspace,
+		localDaemon:       localDaemon,
+		imageIsLocal:      imageIsLocal,
+		structureTestArgs: tc.StructureTestArgs,
```

Comment:
I think it might be better to move this up below line 51 with the `structureTests` to keep the parameters together.  `localDaemon` and `imageIsLocal` are internal matters.